### ISSUE TITLE
Add new card title truncated 

### ISSUE THREAD

**Issue by youphyun** [stefan-niedermann/nextcloud-deck#1385]
Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add!

**Describe the bug**
The Add new card title is truncated on a device with a smaller resolution.

1. Go to 'Card deck'
2. Click on '+' to create a new card
3. View title input field (the title is not even visible if the keyboard is expanded)

**Expected behavior**
Title is fully visible.

**Screenshots**
![Screenshot_20221114-133626_Nextcloud_Deck_s](https://user-images.githubusercontent.com/12627719/201679434-2af8b182-894b-4278-8f0a-2c7d16f1273e.png)
![Screenshot_20221114-133637_Nextcloud_Deck_s](https://user-images.githubusercontent.com/12627719/201681208-3dceb738-bbf8-45d6-ac3f-17ef6ca87eb4.png)

**Versions**
 - Nextcloud: Nextcloud Hub II 23.0.5
 - Nextcloud Deck: 1.6.3
 - Nextcloud Android: 11
 - Nextcloud Android Deck: 1.21.1

**Smartphone (please complete the following information):**
- Device: Samsung Galaxy S5 (klte)
- Android-Version: Latest LineageOs 18.1 (September 25, 2022) ROM Samsung Galaxy S5 (klte) with FlorisBoard keyboard
- App-Store:
  - [ ] Google Play Store
  - [ ] Google Play Store (Beta channel)
  - [x] F-Droid
  - [ ] Huawei AppGallery

**Comment by stefan-niedermann** [comment]
Does this also hapoen eith the default keyboard?  
Which size does your phone screen have?

**Comment by stefan-niedermann** [comment]
*Might* be related to https://github.com/material-components/material-components-android/issues/1480 - we don't use the mentioned code snippet though:
```java
dialog.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_VISIBLE);
```

**Comment by stefan-niedermann** [comment]
@youphyun @Toromtomtom can you verify that this behavior only started with `1.21.0` and it worked properly before that version?

**Comment by Toromtomtom** [comment]
@stefan-niedermann Yep, confirming. I downgraded to 1.20.3, and the input fields work as expected; upgraded to 1.21.1 and they are truncated again.

**Comment by youphyun** [comment]
The Samsung Galaxy S5 has a screen size of 5.1" and a resolution of: 1080x1920 pixels
I also can add that the same issue is present in the Nextcloud Notes app as well.

**Comment by stefan-niedermann** [comment]
Yeap. Looks like Material 3 Dialogs are only designed for "choosable" stuff which do not require any keyboard interaction.

The proposed solution is to use a fullscreen dialog according to the docs. Will nees some time to check out how those are implemented though.

For reference: https://m3.material.io/components/dialogs/specs

**Comment by stefan-niedermann** [comment]
Same issues for official Nextcloud files app and Nextcloud notes app.

**Workaround:** Copy text to clipboard, close keyboard when dialog pops up and paste the text into the input field (direct long press).

**Comment by stefan-niedermann** [comment]
Possible fix https://github.com/nextcloud/notes-android/issues/1660#issuecomment-1409976036

**Comment by stefan-niedermann** [comment]
Yep. Tested the solution and it works. We will ship the fix with the next version.

Thank you for being patient.

**Comment by stefan-niedermann** [comment]
Fix was published in `1.22.0`
